### ISSUE TITLE
Package does not include source

### ISSUE THREAD

**Issue by SamMousa** [surveyjs/custom-widgets#335]
The packages on NPM do not include the source, this makes it hard to use them in frontend projects!

**Comment by SamMousa** [comment]
Interestingly, the latest version: v1.9.133 does contain the source, but it no longer contains the compiled assets. I think this is a bug that will break stuff for most users since you cannot simply import the files as explained in the readme.

@JaneSjs

**Comment by SamMousa** [comment]
There was a change here for sure. I use the git source, but soon i can drop this because of native sliders

**Comment by dmitry-kurmanov** [comment]
> There was a change here for sure. I use the git source, but soon i can drop this because of native sliders

@SamMousa could you please provide two links with sources (e.g. v1.9.133) and without them? Probably, I suppose that I don't fully understand the problem.

**Comment by SamMousa** [comment]
No, given that i'm the only one who's running into it (apparently), and I have a work around in place and I'm hoping to phase out use of these widgets soon I'm not going to look for a the specific release of 2 years ago that changes the release format.
